easypermissions/src/test/java/pub/devrel/easypermissions/EasyPermissionsLowApiTest.java

```diff
@@ -105,7 +105,7 @@ public void shouldCallbackOnPermissionGranted_whenRequestFromFragment() {
     private void setUpActivityAndFragment() {
```

Comment:
I think you should create a controller for the TestSupportFragmentActivity.class to test it here with the low API tests (same as you did that for the high API tests).